packages/app/playwright/crx/crx.test.ts

```diff
@@ -645,6 +668,32 @@ test.describe('FuelWallet Extension', () => {
           'address is not authorized for this connection.'
         );
       });
+
+      await test.step('Signed personalSign string message', async () => {
+        const authorizedAccount = await switchAccount(popupPage, 'Account 1');
+        // Convert hex string to bytes array like the sample dApp does
+        const hexString =
+          '0x3c54688d3e13800c0a6ad3ccf06a67bae8702b5e326ce991b8245d60c671a85a';
+        const hashBytes = new Uint8Array(
+          Buffer.from(hexString.slice(2), 'hex')
+        );
+        const personalStringMessage = { personalSign: hashBytes };
```

Comment:
```suggestion
        const personalStringMessage = { personalSign: hexString };
```

Should we be passing the `hexString` here, given the test name mentions a string message?